controls/collectionview/accessibility/keyboard-navgation/maccatalyst.md

```diff
@@ -0,0 +1,42 @@
+---
+title: MacCatalyst
+page_title: .NET MAUI CollectionView Documentation - Keyboard Support on MacCatalyst
+description: Learn more about the available keyboard combinations as part of the supported Telerik UI for .NET MAUI CollectionView accessibility standards.
+position: 2
+slug: collectionview-keyboard-support-mac
+---
+
+# .NET MAUI CollectionView Keyboard Support on MacCatalyst
+
+The [Telerik UI for .NET MAUI CollectionView]({%slug collectionview-overview%}) provides keyboard navigation support on `MacCatalyst`.
+
+@[template](/_contentTemplates/common/collectionview.md#collectionview-keyboard-common-text)
+
+
+| Hotkey Combinations  | Action |
```

Comment:
"Hotkey" suggests that it could also be a combination of keys, so you can skip "combination".
```suggestion
| Hotkey | Action |
```